paragraphs/breol_link_paragraph/breol_link_paragraph.install

```diff
@@ -24,3 +24,36 @@ function breol_link_paragraph_update_7001() {
     }
   }
 }
+
+/**
+ * Migrate color values.
+ */
+function breol_link_paragraph_update_7002() {
+  $color_map = [
```

Comment:
One could have move this color map and the default color into defined constant breol_link_paragraph, so when we need to change them next week it's in one place.

Reply:
I'll do that next week, @cableman …